sql-plugin/src/main/spark352/scala/com/nvidia/spark/rapids/spark352/RapidsShuffleManager.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
Isn't this a new file?
```suggestion
 * Copyright (c) 2024, NVIDIA CORPORATION.
```